MTGSDKSwift/MTGAPIManager.swift

```diff
@@ -8,89 +8,33 @@
 
 import Foundation
 
+public typealias JSONResults = [String:Any]
+public typealias JSONCompletionWithError = (Result<JSONResults>) -> Void
 
-public typealias JSONResults = [String: Any]
-public typealias JSONCompletionWithError = (JSONResults?, NetworkError?) -> Void
-
-final class URLManager {
-    
-    var pageSize = "6"
-    var pageTotal = "1"
-    
-    
-    func buildURL(parameters: [SearchParameter]) -> URL? {
-        var urlComponents = URLComponents()
-        urlComponents.scheme = "https"
-        urlComponents.host = "api.magicthegathering.io"
-        urlComponents.path = {
-            if parameters is [CardSearchParameter] {
-                return "/v1/cards"
-            } else {
-                return "/v1/sets"
-            }
-        }()
-        
-        urlComponents.queryItems = buildQueryItemsFromParameters(parameters)
-        
-        if Magic.enableLogging == true, let url = urlComponents.url {
-            print("URL: \(url)\n")
-        }
-        
-        return urlComponents.url
-    
-    }
-    
-    private func buildQueryItemsFromParameters(_ parameters: [SearchParameter]) -> [URLQueryItem] {
-        var queryItems = [URLQueryItem]()
-        
-        let pageSizeQuery = URLQueryItem(name: "pageSize", value: pageSize)
-        let pageQuery = URLQueryItem(name: "page", value: pageTotal)
-        queryItems.append(pageQuery)
-        queryItems.append(pageSizeQuery)
-        
-        for parameter in parameters {
-            let name = parameter.name
-            let value = parameter.value
-            let item = URLQueryItem(name: name, value: value)
-            queryItems.append(item)
-        }
-        
-        return queryItems
-    }
-    
+public enum Result<T> {
+    case success(T)
+    case error(NetworkError)
 }
 
-
-
 final class MTGAPIService {
-    
+    /**
```

Comment:
Can we use the newer: `///` style comments?  

Pro tip: To generate them, select the function line that you want the generated header on and type: `Cmd + Option + /` and it will automatically generate it (for example here):
```
/// <#Description#>
///
/// - Parameters:
///   - application: <#application description#>
///   - launchOptions: <#launchOptions description#>
/// - Returns: <#return value description#>
func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```